frontend/app/services/benefit-application-service.server.ts

```diff
@@ -105,22 +106,153 @@ function createBenefitApplicationService() {
+                        }),
+                      }),
+                    ),
+                  }),
+                ),
+                PersonLanguage: z.array(
+                  z.object({
+                    CommunicationCategoryCode: z.object({
+                      ReferenceDataID: z.string(),
+                    }),
+                    PreferredIndicator: z.boolean(),
+                  }),
+                ),
+                PersonMaritalStatus: z.object({
+                  StatusCode: z.object({
+                    ReferenceDataID: z.string(),
+                  }),
+                }),
+                PersonName: z.array(
+                  z.object({
+                    PersonGivenName: z.array(z.string()),
+                    PersonSurName: z.string(),
+                  }),
+                ),
+                PersonSINIdentification: z.object({
+                  IdentificationID: z.string(),
+                }),
+                RelatedPerson: z.array(
+                  z.object({
+                    PersonBirthDate: z.object({
+                      dateTime: z.string(),
+                    }),
+                    PersonName: z.array(
+                      z.object({
+                        PersonGivenName: z.array(z.string()),
+                        PersonSurName: z.string(),
+                      }),
+                    ),
+                    PersonRelationshipCode: z.object({
+                      ReferenceDataName: z.string(),
+                    }),
+                    PersonSINIdentification: z.object({
+                      IdentificationID: z.string(),
+                    }),
+                    ApplicantDetail: z.object({
+                      ConsentToSharePersonalInformationIndicator: z.boolean().optional(),
+                      AttestParentOrGuardianIndicator: z.boolean().optional(),
+                      PrivateDentalInsuranceIndicator: z.boolean().optional(),
+                      InsurancePlan: z
+                        .object({
+                          InsurancePlanIdentification: z
+                            .object({
+                              IdentificationID: z.string().optional(),
+                            })
+                            .array()
+                            .optional(),
+                        })
+                        .array()
+                        .optional(),
+                    }),
+                  }),
+                ),
+                MailingSameAsHomeIndicator: z.boolean(),
+
+                PreferredMethodCommunicationCode: z.object({
+                  ReferenceDataID: z.string(),
+                }),
+              }),
+              BenefitApplicationCategoryCode: z.object({
+                ReferenceDataID: z.string(),
+              }),
+              BenefitApplicationChannelCode: z.object({
+                ReferenceDataID: z.string(),
+              }),
+            }),
+          }),
+        ),
```

Comment:
Can we remove this schema and use the one defined in `application-history-mapper` instead?

Reply:
Moved to `application-history-service-schemas.server`:  9d9c89ece1890dd3d2e16558687fce817737ca11.